eslintrc.template

```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
   "extends": [
-      "skyscanner"
+      "@skyscanner/skyscanner"
```

Comment:
From looking at other packages and usage shouldn't this be `@skyscanner/eslint-config-skyscanner`?

Reply:
Hmm, so we need to update https://github.com/Skyscanner/eslint-config-skyscanner/blob/main/main.js#L93 too?

Reply:
And https://github.com/Skyscanner/eslint-config-skyscanner/blob/main/README.md#installation

Reply:
Interesting testing locally it appears to work either way around with the same output 🤔

Reply:
I like `@skyscanner/eslint-config-skyscanner` more, will push a commit that changes all references.